### ISSUE TITLE
Screen reader cannot click the control bar icons using the keyboard (Accessibility Issue)

### ISSUE THREAD

**Issue by sethcarlton** [jwplayer/jwplayer#2035]
### Expected Behavior

When I am using a screen reader (such as macOS VoiceOver Utility), I should be able to tab to a specific button and "click" it using the keyboard.

### Actual Behavior

Upon using the keyboard command, a click event is fired in the client, but it does not activate the jwplayer button. Do the control bar buttons not have a "click" event registered?

### Steps to reproduce

1. Start a screen reader software
2. Navigate to one of the buttons on the jwplayer control bar
3. Use the appropriate keyboard command to "click" the button (with VoiceOver it is command + option + space)

### Environment

- iMac (27-inch, Late 2013)
- macOS Sierra | 10.12.1
- Google Chrome | 57.0.2987.133
- 7.2.4, 7.11.0 (I think all versions)

**Comment by johnBartos** [comment]
@sethcarlton Hey sorry for the delay - thanks for the report. We're aware of this issue and @hghazzi will be fixing it this quarter. There's a bit of tech debt around keyboards that we need to clean up to make this work. When you say "navigate to one of the buttons" do you mean with the tab key?

**Comment by sethcarlton** [comment]
Thanks @johnBartos! Appreciate the response. As far as the navigation bit, yes, I was referring to the tab key. Sorry for not clarifying that initially.

I'll be on the lookout for this fix in the near future.

**Comment by hghazzi** [comment]
@sethcarlton thanks for bringing this up. It is definitely something I'd like to get to correcting. Will keep you updated when it is ready to be released.

**Comment by johnBartos** [comment]
@sethcarlton  We have a ticket groomed in our backlog and target for `7.12`

**Comment by raghavkamran** [comment]
@sethcarlton We were waiting on getting the fix for this issue in v7.12.0, but looks likes it is still not fixed. Do you have a timeline by when this would be fixed?

**Comment by sethcarlton** [comment]
@raghavkamran we are still waiting for this to be resolved as well. @johnBartos @hghazzi Do you have any updates?

**Comment by sethcarlton** [comment]
Also if it helps, using voice over on a mac with Safari doesn't have this issue.

**Comment by hghazzi** [comment]
@sethcarlton @raghavkamran Sorry for not getting back to you, we've been working hard on getting the next major version of JW Player out the door. Part of the work is improving accessibility support. You will be able to "click" any controlbar button using the `Enter` key. The `Space` key will play/pause the video, no matter where you are on the player. Will these changes give you the functionality you need?

**Comment by sethcarlton** [comment]
@hghazzi - thanks for the update. I have been following along with the JW Player 8 updates and it is looking nice! Using enter to "click" the buttons is working well. I would like to mention that in the newest release candidate (3) the enter key is not working on some of the settings menu items. Captions are working fine, but speed and video quality are not. 

The only issue I am still seeing is that the keyboard command used to click items while using macOS Voice Over (Ctrl + Option + Space) is still not "clicking" them. We should be able to work around this for now but I wanted to make sure you are aware of the issue as it could be confusing for some users.

Hope this helps!

**Comment by johnBartos** [comment]
@sethcarlton Just noticed the enter key issue on certain submenus. Will open a ticket

**Comment by hghazzi** [comment]
@sethcarlton the menu items issue will be fixed in `v8.0.1`. If everything looks good to you otherwise, please close out the ticket. Thanks!

**Comment by sethcarlton** [comment]
Thanks guys!
